app/views/organisations/_show_organisation.html.erb

```diff
@@ -23,19 +23,39 @@
   </div>
 <% end %>
 
+<%= render "govuk_publishing_components/components/heading", {
+  text: "What #{@organisation.title} is doing",
+  brand: @organisation.brand,
+  padding: true,
+  border_top: 5,
+  margin_bottom: 4
+} %>
+
+<% if @organisation.is_news_organisation? %>
+  <%= render "govuk_publishing_components/components/image_card", @featured_news.first_featured_news %>
+<% end %>
+
+<div class="grid-row">
+  <% @featured_news.remaining_featured_news.each_with_index do |news, i| %>
+    <div class="column-one-third">
+      <%= render "govuk_publishing_components/components/image_card", news %>
+    </div>
+
+    <% if i % 3 == 2 %>
```

Comment:
I think this could perhaps be simplified by using `.in_groups_of(3)` to process the news items in batches of 3 instead of having to do the calculations with the index. Something like:

```
<% @featured_news.remaining_featured_news.in_groups_of(3) do |news| %>
  <div class="grid-row">
    <% news.each do |news_item| %>
      <div class="column-one-third">
        <%= render "govuk_publishing_components/components/image_card", news_item %>
      </div>
    <% end %>
  </div>
<% end %>
```

Reply:
Didn't know that was a thing, much better, thanks.